cfs-require-import.php

```diff
@@ -8,32 +8,50 @@
 Author URI: https://www.vanpattenmedia.com/
 */
 
-class CfsRequireImport {
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
 
+class CfsRequireImport {
+	
 	function __construct() {
-		register_activation_hook( plugin_basename( __FILE__ ), array( $this, 'import_cfs_fields' ) );
+		
+		/* Register activation hook. */
+		register_activation_hook( plugin_basename( __FILE__ ), array( $this , 'import_cfs_fields' ) );
+		
+		/* Add admin notice */
+		add_action( 'admin_notices' , array( $this , 'print_import_notice' ) );
 	}
-
+	
+ 	/* Runs only when the plugin is activated. */
```

Comment:
This should be converted to a [docblock](http://docs.phpdoc.org/guides/docblocks.html)! I'm happy to do this if you aren't familiar with that.

Reply:
Please, convert it, i'm new to these things :)